src/processors/file-handler.ts

```diff
@@ -47,25 +50,25 @@ const JSON_LOADER_OPTIONS = {
   ]
 };
 
-export function isGeoJson(json) {
+export function isGeoJson(json: any): boolean {
```

Comment:
you can use type guard here
`isGeoJson(json: unknown): json is Feature | FeatureCollection`

Reply:
fixed